charms/istio-pilot/lib/charms/istio_pilot/v0/istio_gateway_info_receiver.py

```diff
@@ -0,0 +1,55 @@
+import logging
```

Comment:
With charmcraft libraries , each file is a new library. Given that we want to implement one library for this interface, both of them should be in the same file.